index.html

```diff
@@ -7,7 +7,7 @@
     <title>ASOCIATIA OPORTUNITATI SI CARIERE</title>
   </head>
   <body>
-    <header>
+    <header id="top-redirect">
       <nav class="nav">
```

Comment:
Please add a subtle line under the navbar to differentiate the navbar of the rest of the background.